bigquery/google/cloud/bigquery/table.py

```diff
@@ -1026,7 +1026,7 @@ def _configure_job_metadata(metadata,  # pylint: disable=too-many-arguments
         load_config['writeDisposition'] = write_disposition
 
 
-def _parse_schema_resource(info):
+def parse_schema_resource(info):
```

Comment:
Can we punt on the "make this public" discussion? Public interfaces aren't easy to walk away from and it's certainly not the point of this PR.